senza/exceptions.py

```diff
@@ -11,3 +11,12 @@ class VPCError(SenzaException, AttributeError):
 
     def __init__(self, message):
         super().__init__(message)
+
+
+class PiuNotFound(SenzaException, FileNotFoundError):
```

Comment:
Because subprocess.call raises a FileNotFoundError when the executable doesn't exist, and I'm just making it more specific.